src/components/dashboard/SplineGraph.tsx

```diff
@@ -3,16 +3,83 @@ import {
   Button,
   ButtonGroup,
   Grid,
+  LinearProgress,
   Typography,
```

Comment:
We have to use BodyText, Display and Label from our Typography Library, and NOT use Material UI's Typography compoennt